<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectFile</class>
 <widget class="QDialog" name="ProjectFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project File</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mTabPathsAndDefines">
      <attribute name="title">
       <string>Paths and Defines</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Import Project (Visual studio / compile database)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="mEditImportProject">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="mBtnClearImportProject">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <activeon>:/images/edit-clear.png</activeon>
               </iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="mBtnBrowseImportProject">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="mChkAllVsConfigs">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You have a choice:&lt;/p&gt;&lt;p&gt; * Analyze all Debug and Release configurations&lt;/p&gt;&lt;p&gt; * Only analyze the first matching Debug configuration&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Analyze all Visual Studio configurations</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="mLayoutCheckPaths">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="mLabelCheckPaths">
             <property name="text">
              <string>Paths:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="mListCheckPaths"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="mLayoutCheckPathsButtons">
           <item>
            <widget class="QPushButton" name="mBtnAddCheckPath">
             <property name="text">
              <string>Add...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mBtnEditCheckPath">
             <property name="text">
              <string>Edit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mBtnRemoveCheckPath">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="mLayoutDefines">
         <item>
          <widget class="QLabel" name="mLabelDefines">
           <property name="text">
            <string>Defines:</string>
           </property>
           <property name="buddy">
            <cstring>mEditDefines</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mEditDefines">
           <property name="toolTip">
            <string>Defines must be separated by a semicolon. Example: DEF1;DEF2=5;DEF3=int</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="mLayoutIncludePaths">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QLabel" name="mLabelIncludePaths">
             <property name="text">
              <string>Include Paths:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="mListIncludeDirs">
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="mLayoutIncludePathsButtons">
           <item>
            <widget class="QPushButton" name="mBtnAddInclude">
             <property name="text">
              <string>Add...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mBtnEditInclude">
             <property name="text">
              <string>Edit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mBtnRemoveInclude">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="mBtnIncludeUp">
             <property name="text">
              <string>Up</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mBtnIncludeDown">
             <property name="text">
              <string>Down</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mTabChecking">
      <attribute name="title">
       <string>Checking</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Cppcheck build dir (whole program analysis, incremental analysis, statistics, etc)</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="mEditBuildDir"/>
          </item>
          <item>
           <widget class="QPushButton" name="mBtnBrowseBuildDir">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Platform</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QComboBox" name="mComboBoxPlatform"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Libraries</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="mLayoutLibraries"/>
          </item>
          <item>
           <widget class="QLabel" name="mLabelLibrariesNote">
            <property name="text">
             <string>Note: Put your own custom .cfg files in the same folder as the project file. You should see them above.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>96</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mTabWarningOptions">
      <attribute name="title">
       <string>Warning options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Root path:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QLineEdit" name="mEditProjectRoot"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Warning tags (separated by semicolon)</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="mEditTags"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Exclude paths</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QListWidget" name="mListExcludedPaths"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="layoutExcludePathButtons">
            <item>
             <widget class="QPushButton" name="mBtnAddIgnorePath">
              <property name="text">
               <string>Add...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="mBtnEditIgnorePath">
              <property name="text">
               <string>Edit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="mBtnRemoveIgnorePath">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Suppressions</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QListWidget" name="mListSuppressions"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QPushButton" name="mBtnAddSuppression">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="mBtnRemoveSuppression">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_10">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>96</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mTabAddonsAndTools">
      <attribute name="title">
       <string>Addons and tools</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QGroupBox" name="mGroupBoxAddons">
         <property name="title">
          <string>Addons</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="mAddonY2038">
            <property name="text">
             <string>Y2038</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mAddonThreadSafety">
            <property name="text">
             <string>Thread safety</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Coding standards</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mAddonCert">
            <property name="text">
             <string>Cert</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mAddonMisra">
            <property name="text">
             <string>MISRA C 2012</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mGroupBoxTools">
         <property name="title">
          <string>External tools</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_1">
          <item>
           <widget class="QCheckBox" name="mToolClangTidy">
            <property name="text">
             <string>Clang-tidy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mToolClangAnalyzer">
            <property name="text">
             <string>Clang analyzer</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>368</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="mButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mButtons</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mButtons</sender>
   <signal>accepted()</signal>
   <receiver>ProjectFile</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mButtons</sender>
   <signal>rejected()</signal>
   <receiver>ProjectFile</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
